cmd/template/cluster/provider/templates/aws/template.go

```diff
@@ -7,6 +7,9 @@ import (
 //go:embed cluster.yaml.tmpl
 var cluster string
 
+//go:embed cluster_eks.yaml.tmpl
+var cluster_eks string
```

Comment:
Is there a reason this uses `_` instead of camel case?

Reply:
Just that `ClusterEKS` is not the object kind. It's just a `Cluster` but the template is different.
I'm fine with using camel case, but I thought it could make it look like that.

Reply:
I've no strong opinions either way though I think as all the other vars here use camel case having this different suggests it has some sort of different purpose or meaning maybe?

Reply:
let's go with the camel then :small_airplane: